packages/opentelemetry-test-utils/src/test-utils.ts

```diff
@@ -141,11 +141,22 @@ export interface TimedEvent {
   name: string;
   /** The attributes of the event. */
   attributes?: SpanAttributes;
+  /** Count of attributes of the event that were dropped due to collection limits */
+  droppedAttributesCount?: number;
```

Comment:
Reviewer note: This was added to fix test-all-versions tests of ioredis@4 -- a code path not tested with `npm test`. See https://github.com/open-telemetry/opentelemetry-js-contrib/pull/1771/commits/78e64f1cf18ba38946eddc2ce8b5862d6b384b18